README.md

```diff
@@ -48,10 +51,10 @@ jupyter nbextension enable toc2/main
 
 ## Use
 
-The notebook itself contains detailed usage instructions. 
+The notebooks themselves contains detailed usage instructions. 
```

Comment:
I think technically it's 'contain' because 'notebooks themselves' is plural but I could be wrong.

Reply:
You are correct ... I stopped editing that sentence too early :)